clients/drcachesim/tests/memref_gen.h

```diff
@@ -34,10 +34,19 @@
 #define _MEMREF_GEN_ 1
 
 #include "../common/memref.h"
```

Comment:
From offline discussion: make sure this builds when used as an exported header in another directory

Reply:
Done.